src/components/POLine/Cost/CostForm.js

```diff
@@ -24,16 +23,33 @@ import {
 } from '../const';
 import calculateEstimatedPrice from '../calculateEstimatedPrice';
 
+// Validation for Fields with type 'number' requires positive integer
+export const requiredPositiveQuantity = (value) => {
+  return value >= 1
+    ? undefined
+    : <FormattedMessage id="ui-orders.cost.validation.shouldBePositive" />;
+};
+
+const validateRequiredNotNegative = (value) => {
+  return value === 0 || value > 0
```

Comment:
could this be shortened to simply ` value < 0 ? <formattedMessage ...> : undefined`?  Was it done this way for consistency with the other ternary expressions used in the other validation functions?

Reply:
basically in JS the way you've suggested is doing a quite different comparison. we are not sure if value is a `Number` type. So empty string will pass through as a good value in your case

Reply:
Thanks makes sense now... but doesn't that mean that `'0'` would be considered invalid, while `0` or `'2'` are considered valid?

Reply:
That's true, but in overall `return value === 0 || value > 0` precisely describes what we expect as truthy values, so for readability purposes I prefer to leave it as is. 
If you look into `parse={parseMoney}` field attribute, which calls before validate, it return `Number` or some falsy value (empty string included), what we filter here as not valid. Hope this makes sense!

Reply:
hm, what about `value >= 0`?

Reply:
doesn't work with empty string, but fine for undefined

Reply:
Right, `value === 0 || value > 0` gets us 99% of the way there, the case I mention is just an edge case.  I'm fine with leaving this as-is for now.  Thanks for helping me understand guys